assets/css/tablesorter.theme.glotpress.css

```diff
@@ -19,6 +19,7 @@ GlotPress Theme
 	cursor: pointer;
 	text-align: center;
 	border: 1px solid #eee;
+	padding-right: 20px !important;
```

Comment:
The only problem with that is that we get padding where none is need on some tables, like the translations table.

The reason I used `!important` in the tablesorter CSS is so that the styling is associated with only the tables that the tablesorter is active on.  It's one of the few cases where `!important` can be used correctly (IMO of course).

Otherwise we'll have to either split the class definitions for the tables or create a second class just for the right padding to override the override.